build/infrastructure/main/databricks-workspace.tf

```
@@ -0,0 +1,41 @@
+# Copyright 2020 Energinet DataHub A/S
+#
+# Licensed under the Apache License, Version 2.0 (the "License2");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+resource "azurerm_databricks_workspace" "dbw_aggregations" {
```

This workspace is shared and not solely for aggregations.
```suggestion
resource "azurerm_databricks_workspace" "dbw_shared" {
```